resources/watcher.yaml

```diff
@@ -35,3 +35,41 @@ spec:
             runAsUser: 65534
           args:
             - --mode=alerts
+---
```

Comment:
Maybe makes sense to bind watcher only to master nodes?

Reply:
Shouldn't really matter much, but we can do it.